pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -264,6 +267,11 @@ func (r *ReconcileAccountClaim) setSupportRoleARNManagedOpenshift(reqLogger logr
 	if accountClaim.Spec.STSRoleARN == "" {
 		instanceID := account.Labels[awsv1alpha1.IAMUserIDLabel]
 		accountClaim.Spec.SupportRoleARN = fmt.Sprintf(awsv1alpha1.ManagedOpenShiftSupportRoleARN, account.Spec.AwsAccountID, instanceID)
+
+		// if account if fedramp use the appropriate iam arn
+		if config.IsFedramp() {
+			accountClaim.Spec.SupportRoleARN = fmt.Sprintf(awsv1alpha1.FedrampManagedOpenShiftSupportRoleARN, account.Spec.AwsAccountID, instanceID)
+		}
```

Comment:
jprobably you could make something like `config.GetSupportRoleArn()` similar to `getDefaultRegion()`